Dockerfile

```diff
@@ -1,13 +1,23 @@
 FROM alpine:3.6
 
+ENV ETCD_VERSION 3.1.4
+ENV KUBE_VERSION 1.6.2
+
 RUN apk add --update bash curl docker \
     && rm -rf /var/cache/apk/*
 
 RUN cd /usr/local/bin \
-    && curl -O https://storage.googleapis.com/kubernetes-release/release/v1.6.2/bin/linux/amd64/kubectl \
+    && curl -O https://storage.googleapis.com/kubernetes-release/release/v${KUBE_VERSION}/bin/linux/amd64/kubectl \
     && chmod 755 /usr/local/bin/kubectl
 
-COPY docker-clean.sh k8s-clean.sh /bin/
+RUN cd /tmp \
+    && curl -OL https://github.com/coreos/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz \ 
+    && tar zxf etcd-v${ETCD_VERSION}-linux-amd64.tar.gz \
+    && cp etcd-v${ETCD_VERSION}-linux-amd64/etcdctl /usr/local/bin/etcdctl \
+    && rm -rf etcd-v${ETCD_VERSION}-linux-amd64* \
+    && chmod +x /usr/local/bin/etcdctl
+
+COPY docker-clean.sh k8s-clean.sh etcd-empty-dir-cleanup.sh /bin/
 RUN chmod +x /bin/docker-clean.sh && chmod +x /bin/k8s-clean.sh
```

Comment:
Add `etcd-empty-dir-cleanup.sh` to this line as well

Reply:
What do you mean ? It's already line 20.

Reply:
It's copied but not allowed to execute. We might not need those actually as we run them as `/bin/bash <script>`

Reply:
Oh yes ! 👍